---
# LiDAR message types.
#   0 for PointCloud2 XYZIRT
#   1 for LivoxCustomMessage
#   2 for PointCloud2 XYZ
lidar_types: [0, 1]

lidar_odoms: # odometries here:
  - $(find ia_helic)/input/mechanical.txt
  - $(find ia_helic)/input/solid-state.txt

topics:
  lidars:
    - /rslidar_points
    - /livox/lidar
  imu: /livox/imu

# Set this to local gravity for Livox internal IMU, or 1 for other IMUs.
imu_acc_scale_by: 9.805

ROI:
  min: [-256, -256, -256] # in meters
  max: [256, 256, 256] # in meters

ndt_align:
  enable: false
  voxel_size: 0.5 # in meters

optimization:
  surfel_size: 0.5 # in meters
  down_sample_rate: 100
  iterations: 3
  solver_max_iterations: 50
  kont_distance: 0.02 # in seconds
  time_padding: 0 # in seconds
  down_sample_farthest: false
  acc_weight: 18.5
  gyro_weight: 28.0
  lidar_weight: 2.0


calib_file_path: $(find ia_helic)/result/calib_result_iter{iter}.yml

